README.md

```diff
@@ -1 +1,83 @@
-# DBSClient
\ No newline at end of file
+[![Linux](https://svgshare.com/i/Zhy.svg)](https://svgshare.com/i/Zhy.svg) [![macOS](https://svgshare.com/i/ZjP.svg)](https://svgshare.com/i/ZjP.svg)
+# DBS3-Client
+The **Data Bookkeeping Service 3** (DBS 3) provides an improved event data catalog for Monte Carlo and recorded data of the CMS experiment. It 
+provides the necessary information used for tracking datasets, for example the data processing history, files and runs associated with a given 
+dataset, and much more. DBS 3 has been completely re-designed and re-implemented in Python on the basis of a CherryPy based environment for 
+developing RESTful (Representational State Transfer) web services, which is commonly used within the data management and workload management (DMWM) 
+group of CMS. DBS 3 is using the Java Script Object Notation (JSON) dataformat for interchanging information and Oracle as database backend.
+# 1. Install Python
+In this first part we are going to install Python 3 and pip. We are going to follow the following 
+[tutorial](https://thelazylog.com/install-python-as-local-user-on-linux/) in the next sections.
+## 1.1 Installation of Python 3.8.2
+We are going to install Python 3.6.5. *Note : You can download any version higher that 3.6.0 instead.*
+    mkdir ~/python
+    cd ~/python
+    wget https://www.python.org/ftp/python/3.8.2/Python-3.8.2.tgz
+    tar zxfv Python-3.8.2.tgz
+    find ~/python -type d | xargs chmod 0755
+    cd Python-3.8.2
+    ./configure --prefix=$HOME/python If you get an "No acceptable C compiler found in $PATH" error while running the last command, it mean that 
+python could not find gcc. Please then run the following command: - Redhat base: `yum groupinstall "Development Tools"` - Debian base: `apt-get 
+install build-essential` Solution found on 
```

Comment:
you need to add below:
sudo yum install libcurl-devel
sudo yum install openssl-devel
sudo yum -y install libffi-devel
and check the google doc for my comments. You closed some the comments without applying them.